c2cgeoportal/lib/lingua_extractor.py

```
@@ -68,20 +69,36 @@ def __call__(self, filename, options):
 
 
 class GeoMapfishConfigExtractor(Extractor):  # pragma: nocover
-    "GeoMapfish config extractor (raster layers)"
+    "GeoMapfish config extractor (raster layers, and print templates)"
 
     extensions = [".yaml"]
 
     def __call__(self, filename, options):
         with open(filename) as config_file:
             config = yaml.load(config_file)
-            return [
-                Message(
-                    None, raster_layer, None, [], u"", u"",
-                    (filename, u"raster/%s" % raster_layer)
-                )
-                for raster_layer in config.get("raster", {}).keys()
-            ]
+            if "raster" in config:
+                return [
+                    Message(
+                        None, raster_layer, None, [], u"", u"",
+                        (filename, u"raster/%s" % raster_layer)
+                    )
+                    for raster_layer in config.get("raster", {}).keys()
+                ]
+            elif "templates" in config:
+                result = []
+                for template in config.get("templates").keys():
+                    result.append(Message(
+                        None, template, None, [], u"", u"",
+                        (filename, u"template/%s" % template)
+                    ))
+                    result += [
+                        Message(
+                            None, attribute, None, [], u"", u"",
+                            (filename, u"template/%s/%s" % (template, attribute))
+                        )
+                        for attribute in config.get("templates")[template].attributes.keys()
+                    ]
+                return result
 
```

Maybe you don't want to silently ignore files you don't understand:
```
else:
    goBoom!
```